Cargo.toml

```diff
@@ -18,7 +18,7 @@ binary-install = "0.0.3-alpha.1"
 chrome-devtools-rs = { version = "=0.0.0-alpha.1", features = ["color"] }
 chrono = "0.4.19"
 clap = "2.33.3"
-cloudflare = "0.6.6"
+cloudflare = { git = "https://github.com/cloudflare/cloudflare-rs", branch = "cass/fix-tail-response" }
```

Comment:
😄  safe to assume this will be merged upstream and changed here prior to us releasing?

Reply:
@caass , you should be able to remove this and just go with the latest release `0.8.0`. That bugfix doesn't have to do with these changes so they don't need to go through together.

Reply:
@nataliescottdavidson i'm actually just about to release 0.8.1 with these changes 🙈